docs/cow-protocol/tutorials/cow-explorer/address.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+id: address
+---
+
+# By user address
+
+A user may want to see the details of all orders that they have placed. When searching for their address, they will be presented with an overview of all orders that they have placed.
+
+For example, below the user has searched for their address [`0x2557...3af7`](
+https://explorer.cow.fi/gc/address/0x2557ed03e34f0141722a643589f007836a683af7) that contains multiple orders that have been placed on CoW Protocol on Gnosis Chain:
+
+![User address overview](/img/explorer/address_overview.png)
+
+By default, the explorer will show all orders that have been placed by the user, regardless of their status, in reverse chronological order. The user can filter the results by status, and sort the results by any of the columns.
```

Comment:
> The user can filter the results by status, and sort the results by any of the columns.

Not really.
Not possible to filter nor sort in this view.

```suggestion
By default, the explorer will show all orders that have been placed by the user, regardless of their status, in reverse chronological order.
```

Reply:
Thanks for picking up the deliberate mistake :sweat_smile: